examples/aws/10-deployment.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 the Heptio Ark contributors.
+# Copyright 2018 the Heptio Ark contributors.
```

Comment:
Since this is just a straight move, I don't think there's a need to change the date.